nextflow.config

```diff
@@ -250,12 +255,12 @@ dag {
 
 manifest {
     name            = 'nf-core/ampliseq'
-    author          = 'Daniel Straub, Alexander Peltzer'
+    author          = """Daniel Straub, Alexander Peltzer"""
     homePage        = 'https://github.com/nf-core/ampliseq'
-    description     = 'Amplicon sequencing analysis workflow using DADA2 and QIIME2'
+    description     = """Amplicon sequencing analysis workflow using DADA2 and QIIME2"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=21.10.3'
-    version         = '2.4.1'
+    nextflowVersion = '!>=22.10.1'
+    version         = '2.4.2'
```

Comment:
```suggestion
    version         = '2.5.0'
```

Reply:
done in https://github.com/nf-core/ampliseq/pull/550